intern/remote/dvid/remote.py

```diff
@@ -0,0 +1,454 @@
+
+		Args:
+			None
+
+		Returns:
+			None
+
+		Raises:
+			(KeyError): if given invalid version.
+		"""
+        volume_cfg = self._load_config_section(CONFIG_VOLUME_SECTION)
+        proto = volume_cfg[CONFIG_PROTOCOL]
+        host = volume_cfg[CONFIG_HOST]
+        api = proto + "://" + host
+
+        self._volume = VolumeService(api)
+        self._volume.base_protocol = proto
+
+    def _init_versioning_service(self):
+        """Method to initialize the Volume Service from the config data
+
+		Args:
+			None
+
+		Returns:
+			None
+
+		Raises:
+			(KeyError): if given invalid version.
+		"""
+        versioning_cfg = self._load_config_section(CONFIG_VERSIONING_SECTION)
+        proto = versioning_cfg[CONFIG_PROTOCOL]
+        host = versioning_cfg[CONFIG_HOST]
+        api = proto + "://" + host
+
+        self._versioning = VersioningService(api)
+        self._versioning.base_protocol = proto
+
+    def __repr__(self):
+        """Stringify the Remote.
+
+		Returns a representation of the DVIDRemote that lists the host.
+		"""
+        return "<intern.remote.DVIDRemote [" + self._config["Default"]["host"] + "]>"
+
+    def _load_config_section(self, section_name):
+        """Method to load the specific Service section from the config file if it
+		exists, or fall back to the default
+
+		Args:
+			section_name (str): The desired service section name
+
+		Returns:
+			(dict): the section parameters
+		"""
+        if self._config.has_section(section_name):
+            # Load specific section
+            section = dict(self._config.items(section_name))
+        elif self._config.has_section("Default"):
+            # Load Default section
+            section = dict(self._config.items("Default"))
+        else:
+            raise KeyError(
+                (
+                    "'{}' was not found in the configuration file and no default "
+                    + "configuration was provided."
+                ).format(section_name)
+            )
+
+        # Make sure section is valid
+        if "protocol" in section and "host" in section:
+            return section
+        else:
+            raise KeyError(
+                "Missing values in configuration data. "
+                + "Must contain: protocol, host"
+            )
+
+    def get_instance(self, UUID, data_instance, datatype=""):
+        """Method to input all data instance hierarchy requirememnts, works as a dummy
+        for DVIDRemote Parallelism.
+
+		Args:
+			UUID (str) : Root UUID of the repository
+			data_instance (str): Name of data instance within repository
+			datatype (str): data type of the instance (uint8, uint16, uint64) defaults to uint8
+		Returns:
+			resource (intern.resource.dvid.DvidResource)
+		"""
+        return DataInstanceResource(data_instance, UUID, datatype=datatype)
+
+    def get_cutout(self, resource, res, xrange, yrange, zrange, **kwargs):
+        """Method to request a volume of data from DVID server
+
+		Args:
```

Comment:
low priority, we can hit later. but these docstrings should have 4 ~8~ space docstrings

Reply:
👍